diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/BlockStructureBraces.kt

```diff
@@ -220,6 +237,7 @@ class BlockStructureBraces : Rule("block-structure") {
     class BlockStructureBracesConfiguration(config: Map<String, String>) : RuleConfiguration(config) {
         val openBrace = config["openBraceNewline"]?.toBoolean() ?: true
         val closeBrace = config["closeBraceNewline"]?.toBoolean() ?: true
+        val emptyBlockNewline = config["emptyBlockNewline"]?.toBoolean() ?: true
```

Comment:
no-no - you should support an option (with enum values for example) that will do the following:
user will be able to choose, how to format blank blocks:
1) fun foo() {}
or 
2) fun foo(){
}

Reply:
and the second inspection - to prohibit empty blocks (you can even fix it) - for if-else, loops and catch